content/applications/general/email_communication/email_template.rst

```diff
@@ -1,56 +1,128 @@
 ===============
-Email Templates
+Email templates
 ===============
 
-We all know writing good emails is vital to get a high response rate, but you do not want to
-rewrite the same structure every time, do you? That is where email templates come in.
-Without the need to rewrite the entire email structure every time, you save time to focus on
-the content. Multiple templates also let you deliver the right message to the right audience,
-improving their overall experience with the company.
+Writing effective emails is vital to receive a high response rate, to make things easier email
+templates allow the user to send emails without having to compose or rewrite the same structure
+every time. Without the need to rewrite the entire email structure every time, time is saved.
+Multiple templates also let the user deliver the right message to the right audience, improving
+their overall experience with the company.
 
 .. note::
-   The email templates use QWeb. The composer allows you to edit emails in their final rendering,
-   making customizations more robust as you don’t have to edit code.
+   The email templates use QWeb or XML. The composer allows the user to edit emails in their final
+   rendering, making customizations more robust as the user doesn't have to edit code.
 
-Defining a default reply to on your mail template
-=================================================
+Editing email templates
+=======================
 
-Although the field *reply to* is available within the mail templates, **this field is only used
-for mass mailing** mode (this means when sending templates on what we call bulk emailing). You
-can send emails in bulk in almost every app that has a list view. Select the records you want
-and click on the action button. If you have an option to send an email, you will see a mail
-composer with possible values to define:
+Starting Odoo 15, the Powerbox feature was released. The Powerbox allows for the user to edit the
+format of the text, to add links or buttons, appointment options, and to add images to the email
+template. Additionally the XML/HTML code of the email template can be edited via the :guilabel:`</>`
+icon. Dynamic placeholders (referencing fields within Odoo) are only available in the subject line
+and to and from fields in Odoo 15.
 
-.. image:: email_template/composer-mass-mailing-quotations.png
+Powerbox feature
+----------------
+
+The Powerbox feature is a enriched texted editor with many different options to format the layout
+and text style or add XML/HTML features in an email template. The Powerbox is activated by typing a
+forward slash `/` in the body of the email template.
+
+The following options will be available once the Powerbox is activated:
+
+**Basic Blocks**
+
+- Three different heading types
+- A text paragraph block
+- A bulleted list
+- A numbered list
+- A checklist
+- A table
+- The ability to switch the direction of the text
+- A horizontal separator
+- A quote
```

Comment:
I would remove the "A" from the start of any bullet that includes it

Reply:
Thanks, this is a good point.